docs/non-normative/groups/system/design-philosophy.md

```diff
@@ -0,0 +1,94 @@
+# System Semantic Conventions: Instrumentation Design Philosophy
+
+The System Semantic Conventions are caught in a strange dichotomy that is unique among other semconv groups. While we want to make sure we cover obvious generic use cases, monitoring system health is a very old practice with lots of different existing strategies. While we can cover the basic use cases in cross platform ways, we want to make sure that users who specialize in certain platforms aren't left in the lurch; if users aren't given recommendations for particular types of data that isn't cross-platform and universal, they may come up with their own disparate ideas for how that instrumentation should look, leading to the kind of fracturing that the semantic conventions should be in place to avoid.
+
+The following sections address some of the most common instrumentation design questions, and how we as a working group have opted to address them. In some cases they are unique to the common semantic conventions guidance due to our unique circumstance, and those cases will be called out specifically.
+
+## Namespaces
+
+Relevant discussions: [\#1161](https://github.com/open-telemetry/semantic-conventions/issues/1161)
+
+The System Semantic Conventions generally cover the following namespaces:
+
+* `system`  
+* `process`  
+* `host`  
+* `memory`  
+* `network`  
+* `disk`  
+* `memory`  
+* `os`
+
+Deciding on the namespace of a metric/attribute is generally informed by the following belief:
+
+**The namespace of a metric/attribute should logically map to the Operating System concept being considered the instrumentation source.**
```

Comment:
```suggestion
**The namespace of a metric/attribute should logically map to the Operating System concept being considered as the instrumentation source.**
```

Reply:
Done in [487af83](https://github.com/open-telemetry/semantic-conventions/pull/1618/commits/487af83f99a9ab8eeeb937e6cfac28638114cd8b)